balloc/balloc.h

```diff
@@ -187,8 +187,9 @@ struct m0_balloc_super_block {
         m0_bcount_t	bsb_groupsize;        /**< group size in blocks */
```

Comment:
It is not related to your patch, but please fix the indentation issue for this struct.
Some of the lines are indented with "TAB" (that is correct). Some of them are using 8 spaces.

Reply:
Done..